### PR TITLE
Lesson 2

### PR DESCRIPTION
**Pull request by MadhuBeesetty**
listing out all names.

### CODE REVIEW

**Review by kinuesantos — APPROVED** [review]

**Review by allopez7 — CHANGES_REQUESTED** [review]
You don't need name.js here. Only except version.

**Review by allopez7 — CHANGES_REQUESTED** [review]
Why do you have two lessons here?

File: lesson-1/test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const m =require('./solu.js');
+const add =(x,y,z)=>{
```
Comment (allopez7) [inline lesson-1/test.js:2]: Change x,y,z to more descriptive names.

File: lesson-2/name_except_me.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const a =require("fs");
+const name =(err,files)=>{
+  let list;
```
Comment (saumya665) [inline lesson-2/name_except_me.js]: make the string empty to initialize

File: lesson-1/solu.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const add =(a,b)=>{
+  const c=(a+b);
+  return c;
```
Comment (allopez7) [inline lesson-1/solu.js:3]: No need for c, just return a+b

File: lesson-2/name_except_me.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  files.forEach((ele)=>{
+    if(ele !== "madhub")
+    list=list+" "+ele;
+  });
+    console.log(list);
```
Comment (saumya665) [inline lesson-2/name_except_me.js]: write the names to a new file. no need to print

File: lesson-2/names.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+const a=require("fs");
+const name =(err,files)=>{
+  console.log(files);
```
Comment (saumya665) [inline lesson-2/names.js]: this file is not needed.

File: lesson-2/names.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+const a=require("fs");
```
Comment (allopez7) [inline lesson-2/names.js]: change a to a more descriptive name.

File: lesson-1/solu.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+const add =(a,b)=>{
```
Comment (allopez7) [inline lesson-1/solu.js:1]: change a and be to more descriptive names.

**Review by allopez7 — CHANGES_REQUESTED** [review]
Changes required.

File: lesson-2/name_except_me.js [hunk 8]
```diff
@@ -1,10 +1,10 @@
-const a =require("fs");
+const readfile =require("fs");
 const name =(err,files)=>{
-  let list;
+  let list=" ";
   files.forEach((ele)=>{
     if(ele !== "madhub")
     list=list+" "+ele;
```
Comment (allopez7) [inline lesson-2/name_except_me.js]: ele is not very clear. What do you mean? Change it to something more clear.

Comment (saumya665) [reply lesson-2/name_except_me.js]: I agree with Alberto

File: lesson-2/name_except_me.js [hunk 9]
```diff
@@ -0,0 +1,12 @@
+const fs =require("fs");
+const name =(err,files)=>{
+  let list=" ";
+  files.forEach((ele)=>{
```
Comment (saumya665) [inline lesson-2/name_except_me.js:4]: use a better variable name than ele

**Review by saumya665 — APPROVED** [review]

**Pull request merged**